packages/oid4vci-demo-frontend/src/pages/SSIWelcomeScreen/index.tsx

```diff
@@ -59,8 +59,8 @@ const SSIWelcomePage: React.FC = (): ReactElement => {
                     }
                 </div>
                 <SSIPrimaryButton
-                    caption={'Continue'}
-                    style={{width: 200, ...style}}
+                    caption={t("label_continue")}
```

Comment:
this should have been fixed already

Reply:
ah i see, here it's still called SSIWelcomeScreen. it is now called: SSIWelcomePage. guess that is why github is not complaining.

you need to sync with develop

Reply:
The caption was fixed after I updated latest develop.